src/style.css

```diff
@@ -1 +1,123 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url('./fonts/Avenir-Book.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  font-style: normal;
+  src: url('./fonts/Avenir-Heavy.ttf') format('truetype');
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 8px;
+  padding: 0;
+  font-size: 16px;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
```

Comment:
Be consistent with your margins to avoid potential margin collapse. It's better to add only top or bottom margins but not both, and in this case, you have set the margin on the `body` which could lead to unexpected results when combined with other margins.